### PR TITLE
Refs [Task][ #450186][Dashboard student component]

### PR DESCRIPTION
**Pull request by conglt10**
image

https://dev.sun-asterisk.com/issues/450186

### CODE REVIEW

File: client/src/_store/student.module.js [hunk 1]
```diff
@@ -11,10 +11,23 @@ const state = {
...
 };
 
 const actions = {
+  async getSummaryInfo({ commit }) {
+    try {
+      let summaryInfo = await studentService.getSummaryInfo();
+      commit('getSummaryInfo', summaryInfo);
+      return summaryInfo;
+    } catch (error) {
+      console.log(error);
```
Comment (ntquang22298) [inline client/src/_store/student.module.js]: log nay a

File: server/routes/me.js [hunk 2]
```diff
@@ -81,6 +81,72 @@ router.get('/', async (req, res) => {
...
+    }
+
+    let data = JSON.parse(response.msg);
+
+    courses = data.Courses;
+    classes = data.Classes;
+
+    return res.json({
+      success: true,
+      numberCourses: courses.length,
```
Comment (dotrungkien) [inline server/routes/me.js]: `courseCount`

File: server/routes/me.js [hunk 3]
```diff
@@ -81,6 +81,72 @@ router.get('/', async (req, res) => {
...
+      numberClasses: classes.length
```
Comment (dotrungkien) [inline server/routes/me.js]: `classCount`

File: server/routes/me.js [hunk 4]
```diff
@@ -81,6 +81,72 @@ router.get('/', async (req, res) => {
...
+        msg: response.msg.toString()
+      });
+    }
+
+    let data = JSON.parse(response.msg);
+    classes = data.Classes;
+
+    return res.json({
+      success: true,
+      numberClasses: classes.length
```
Comment (dotrungkien) [inline server/routes/me.js]: `classCount`

File: client/src/views/student/Dashboard.vue [hunk 5]
```diff
@@ -1,142 +1,51 @@
...
-              />
-            </b-col>
-          </b-row>
-        </div>
-      </div>
+      <h1 class="h3 mb-2 text-gray-800 mt-3">Dashboard</h1>
+      <el-tag type="success" class="mt-5 mb-2"
+        >Number Courses is enrolling: {{ summaryInfo.numberCourses }}</el-tag
+      >
+      <el-tag>Number Classes is enrolling: {{ summaryInfo.numberClasses }}</el-tag>
```
Comment (dotrungkien) [inline client/src/views/student/Dashboard.vue]: `Number of enrolling courses` `Number of enrolling classes`

**Pull request merged**